File: CartControllerLocalPersistence.swift

class CartControllerLocalPersistence {


    var listOrder : [String]?
    
    @IBOutlet weak var searchTextField: UITextField!
    @IBOutlet var resultsTableView : UITableView!

    override func viewDidLoad() {
        super.viewDidLoad()
        searchTextField.addTarget(self, action: "textFieldDidChange:", forControlEvents: UIControlEvents.EditingChanged)
    }
     
	

func  insertDataListOrder(listOrder: [Order])  -> [Order] {
   var x: Int	     
   let objProdDao =  ProductDAO()	     
   listOrderProduct = [Order]()
   let objOrder =  Order()	     
   let objProduct =  Product()	     
   x=0
   while x<listOrder.size {
      objOrder = listOrder.get(x)
      objProduct = objProdDao.SelectSpecificIdObject(objProduct: objProduct, objOrder: objOrder)
      objOrder.name = objProduct.name
      objOrder.price = objProduct.price
      listOrderProduct.append(objOrder: objOrder)	  
   x+=1
   }

   return listOrderProduct
}

}

File: CartControllerLocalPersistence.java

public class CartControllerLocalPersistence {


    View myView;
    
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        
    }	

public Order onViewCreated(Bundle savedInstanceState){
   OrderDAO objOrderDAO = new OrderDAO();
   List<Order> listOrder  = new ArrayList<Order>();
   String ord;
   ord = "Order by id";
   listOrder = objOrderDAO.SelectObject(ord);
   listOrder = insertDataListOrder(listOrder);

        super.onViewCreated(view, savedInstanceState);
        getActivity().setTitle("Order");
        OrderAdapter adapter = new OrderAdapter(
                getActivity(),
                R.layout.Order.item_file,
                listOrder);

        ListView lv = (ListView)getView().findViewById(R.id.Order_list);
        lv.setAdapter(adapter);	
}

public List<Order> insertDataListOrder(List<Order> listOrder) {
   int x;
   ProductDAO objProdDao = new ProductDAO();
   List<Order> listOrderProduct  = new ArrayList<Order>();
   Order objOrder = new Order();
   Product objProduct = new Product();

   for (x=0;x<listOrder.size();x++){
         objOrder = listOrder.get(x);
         objProduct = objProdDao.SelectSpecificIdObject(objProduct, objOrder);
         objOrder.name = objProduct.name;
         objOrder.price = objProduct.price;
         listOrderProduct.add(objOrder);
   }

   return listOrderProduct;
}


}
